src/test/scala/ai/privado/languageEngine/python/PythonS3TaggerTest.scala

```diff
@@ -23,38 +23,18 @@
 
 package ai.privado.languageEngine.python
 
-import ai.privado.cache.{AppCache, DatabaseDetailsCache, RuleCache, S3DatabaseDetailsCache, TaggerCache}
+import ai.privado.cache.{AppCache, DatabaseDetailsCache, RuleCache, S3DatabaseDetailsCache}
 import ai.privado.entrypoint.PrivadoInput
 import ai.privado.exporter.SinkExporter
 import ai.privado.languageEngine.python.config.PythonDBConfigTagger
-import ai.privado.languageEngine.python.passes.PrivadoPythonTypeHintCallLinker
 import ai.privado.languageEngine.python.tagger.PythonS3Tagger
 import ai.privado.model.*
 import ai.privado.tagger.sink.RegularSinkTagger
-import better.files.File
-import io.joern.pysrc2cpg.{
-  ImportsPass,
-  Py2CpgOnFileSystem,
-  Py2CpgOnFileSystemConfig,
-  PythonInheritanceNamePass,
-  PythonTypeRecoveryPassGenerator
-}
-import io.joern.x2cpg.X2Cpg
-import io.joern.x2cpg.passes.base.AstLinkerPass
-import io.joern.x2cpg.passes.callgraph.NaiveCallLinker
-import io.shiftleft.codepropertygraph.generated.Cpg
-import io.shiftleft.semanticcpg.language.*
-import io.shiftleft.semanticcpg.layers.LayerCreatorContext
+import ai.privado.testfixtures.PythonFrontendTestSuite
 import org.scalatest.BeforeAndAfterAll
 import org.scalatest.matchers.should.Matchers
-import org.scalatest.wordspec.AnyWordSpec
-
-import scala.collection.mutable
 
-class PythonS3TaggerTest extends AnyWordSpec with Matchers with BeforeAndAfterAll {
-  private val cpgs                   = mutable.ArrayBuffer.empty[Cpg]
-  private val outPutFiles            = mutable.ArrayBuffer.empty[File]
-  private val inputDirs              = mutable.ArrayBuffer.empty[File]
+class PythonS3TaggerTest extends PythonFrontendTestSuite with Matchers with BeforeAndAfterAll {
```

Comment:
remove `with matches with BeforeAndAfterAll` as that is already part of the test suite